KnightForge/theme.css

```
@@ -0,0 +1,113 @@
+// SPDX-FileCopyrightText: yar2000T <https://github.com/yar2000T>
+// SPDX-License-Identifier: CC-BY-4.0

```

I think `//` is not a valid comment in CSS. And the CC-BY obliges you to name the original author. So this would be correct in this case:

```suggestion
/* SPDX-FileCopyrightText: yar2000T <https://github.com/yar2000T> */
/* SPDX-FileCopyrightText: Simon Schneegans <code@simonschneegans.de> */
/* SPDX-License-Identifier: CC-BY-4.0 */
```